Cargo.toml

```diff
@@ -30,10 +30,10 @@ futures = "0.3.30"
 futures-core = "0.3.30"
 hex = "0.4.3"
 log = "0.4.20"
-openmls = { git = "https://github.com/xmtp/openmls", branch = "main" }
-openmls_basic_credential = { git = "https://github.com/xmtp/openmls", branch = "main" }
-openmls_rust_crypto = { git = "https://github.com/xmtp/openmls", branch = "main" }
-openmls_traits = { git = "https://github.com/xmtp/openmls", branch = "main" }
+openmls = { git = "https://github.com/xmtp/openmls", rev = "0da7dcb" }
```

Comment:
I don't know about the rest of you, but I've been having weird issues with Rust Analyzer caching dependencies very aggressively. Even after running cargo update, it keeps the old version for type checking (but builds work fine when my code uses newer types). This persists even after running `cargo clean`. Pinning to a specific rev breaks the cache.

Reply:
Reloading the workspace and restarting the server also didn't help

Reply:
Woah. Maybe